packages/plugins/plugin-transcription/src/capabilities/app-graph-builder.ts

```diff
@@ -69,17 +58,17 @@ export default (context: PluginsContext) =>
           {
             id: `${fullyQualifiedId(meeting)}/action/start-stop-transcription`,
             data: async () => {
-              const call = context.requestCapability(MeetingCapabilities.CallManager);
-              invariant(state.transcriptionManager);
-              state.enabled = !state.enabled;
-              void state.transcriptionManager.setEnabled(state.enabled);
+              // NOTE: We are not saving the state of the transcription manager here.
```

Comment:
so this is waiting to update the state until you get the call state update from the swarm because otherwise there's a race between you local interaction and the swarm state?

Reply:
Yes